app/portainer/views/init/endpoint/initEndpoint.html

```diff
@@ -77,15 +77,25 @@
                 <div class="col-sm-12">
                   <span class="small">
                     <p class="text-primary">
-                      Manage the Docker environment where Portainer is running using the Unix filesystem socket.
+                      Manage the Docker environment where Portainer is running using the filesystem socket.
```

Comment:
Please change the text to: `Manage the Docker environment where Portainer is running`